### ISSUE TITLE
Email Campaign Instantly Marked As Read

### ISSUE THREAD

**Issue by cyanstar** [mautic/mautic#8548]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
I've recently sent a small segment campaign out to 70 contacts. Upon sending the campaign it marked every single contact as reading the email instantly. There wasn't any error logs. We're using AWS SES to process the sending, could this be causing it? This wasn't an issue before upgrading to v2.16.0.

| Q   | A
| --- | ---
| Mautic version | v2.16.0
| PHP version | 
| Browser | Chrome

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by gucio1200** [comment]
We are struggling same issue .

**Comment by cyanstar** [comment]
@gucio1200 I created a fresh install to solve the issue. Not really sure what caused it. I have noticed that on 1 of my campaigns whilst testing, it did it again. But only on one specific campaign whilst testing the bounce system - so I put it down to the AWS SNS.

**Comment by individual-it** [comment]
@cyanstar what do you mean with a new install? copy all files again, or also a clean DB?

**Comment by RCheesley** [comment]
I think this might be a duplicate of #7884 perhaps?

**Comment by gucio1200** [comment]
I do not think its duplicated, for us its happening not only for gmail.

**Comment by dichvuhuuich** [comment]
We have same issue after upgrade mautic from 2.15.1 to 2.16.0, we still follow this.

**Comment by cyanstar** [comment]
@RCheesley This isn't a duplication of #7884 The issue highlighted there is that Gmail replaces the tracking links causing false positives. The issue here is that it automatically marked everyone as read the email whilst it was still being processed. Within 1 second of sending the email, they were marked as read. I checked the email and only 2 of them had a Gmail account.

I redeployed a clean install of the software and moved the Database over to it. In the previous instance, I updated using the Mautic upgrade function. 

Since moving over to the clean install, instead of updating an already running one, the issue has been resolved. But I think there is an underlying issue somewhere with the update from 2.15 to 2.16. Just not sure where or what.

**Comment by RCheesley** [comment]
When you ran the upgrade from 2.15 to 2.16 did you encounter any errors? Did you run it at command line? If it works fine with a vanilla install of 2.16 it might suggest that there was a failed update, or an update which had not completed fully. Glad to hear you got it sorted anyway.

**Comment by dichvuhuuich** [comment]
> When you ran the upgrade from 2.15 to 2.16 did you encounter any errors? Did you run it at command line? If it works fine with a vanilla install of 2.16 it might suggest that there was a failed update, or an update which had not completed fully. Glad to hear you got it sorted anyway.

I've updated via command line, and no error shows.

**Comment by dichvuhuuich** [comment]
> @RCheesley This isn't a duplication of #7884 The issue highlighted there is that Gmail replaces the tracking links causing false positives. The issue here is that it automatically marked everyone as read the email whilst it was still being processed. Within 1 second of sending the email, they were marked as read. I checked the email and only 2 out of 70 of them had a Gmail account.
> 
> I redeployed a clean install of the software and moved the Database over to it. In the previous instance, I updated using the Mautic upgrade function.
> 
> Since moving over to the clean install, instead of updating an already running one, the issue has been resolved. But I think there is an underlying issue somewhere with the update from 2.15 to 2.16. Just not sure where or what.

After fresh install, how did you moved old Database over to new database?

**Comment by cyanstar** [comment]
> When you ran the upgrade from 2.15 to 2.16 did you encounter any errors? Did you run it at command line? If it works fine with a vanilla install of 2.16 it might suggest that there was a failed update, or an update which had not completed fully. Glad to hear you got it sorted anyway.

Hiya, there wasn't a single error when I ran the update. Nothing in the error logs in Mautic or apache. It was run via the update notification inside mautic. I also ran a schema update after noticing the issue which didn't resolve it. I'd actually suggest that there's a fault in the update via the notification. I've done it now on 2 other installs and they've reproduced the same issue - I have 26 installations of mautic running at the moment, clean installing all of them to upgrade isn't really viable.

> After fresh install, how did you moved old Database over to new database?

I exported and backed up my DB via PHPmyadmin and imported it into the new server. I then ran a Schema Update to make sure it updated into the present version of Mautic. To update your schema go to {yourmautic.url}/s/update/schema

**Comment by dichvuhuuich** [comment]
> > When you ran the upgrade from 2.15 to 2.16 did you encounter any errors? Did you run it at command line? If it works fine with a vanilla install of 2.16 it might suggest that there was a failed update, or an update which had not completed fully. Glad to hear you got it sorted anyway.
> 
> Hiya, there wasn't a single error when I ran the update. Nothing in the error logs in Mautic or apache. It was run via the update notification inside mautic. I also ran a schema update after noticing the issue which didn't resolve it. I'd actually suggest that there's a fault in the update via the notification. I've done it now on 2 other installs and they've reproduced the same issue - I have 26 installations of mautic running at the moment, clean installing all of them to upgrade isn't really viable.
> 
> If I recall, there were a few alterations in 2.16 around the tracking pixels and email handling. I'm wondering if something isn't quite compatible or hasn't been pulled through properly.
> 
> > After fresh install, how did you moved old Database over to new database?
> 
> I exported and backed up my DB via PHPmyadmin and imported it into the new server. I then ran a Schema Update to make sure it updated into the present version of Mautic. To update your schema go to {yourmautic.url}/s/update/schema

I've think that you install fresh Mautic, after that override old database to new fresh mautic database

**Comment by kamikazebr** [comment]
EDITED: I got the same issue. Just fresh install version 2.15.3 from zip file.

Looks like my https certificate was invalid. So fixing it get back working.

**Comment by drptiago** [comment]
Same issue with v3.2.5

**Comment by HappyCustomers** [comment]
> EDITED: I got the same issue. Just fresh install version 2.15.3 from zip file.
> 
> Looks like my https certificate was invalid. So fixing it get back working.

@kamikazebr
How did you fix it? Can you please provide the steps?
Thanks in Advance

**Comment by kamikazebr** [comment]
> > EDITED: I got the same issue. Just fresh install version 2.15.3 from zip file.
> > Looks like my https certificate was invalid. So fixing it get back working.
> 
> @kamikazebr
> How did you fix it? Can you please provide the steps?
> Thanks in Advance

Hi @HappyCustomers , i dont remember the exact steps.

Look the previous conversation. I did like that:

Download the version zip, take in releases page in github.

Unpack and copied the files configs (dont remember the names files now)

Copied cloned the database and inserted in the fresh one.

Ran the update script of mautic.

Sorry for not help and forget the made the steps correctly, but i think i read a lot too in documentation to make it done.

Remember the clear caches too after each step.

Ahh and about the certificates, it need be valid, i make with Lets Encrypt SSL, if you make the certificates first and then start the install the new fresh mautic, mautic will tell if it is correct.
